README.md

```diff
@@ -18,15 +18,12 @@ npm install --save @odyzeo/form-checkbox
 yarn add @odyzeo/form-checkbox
 ```
 
-Import component in your where you want to use it and register it:
+Import component and tell vue to use it as a plugin:
 
 ```
 import FormCheckbox from '@odyzeo/form-checkbox';
-export default {
-  components: {
-    FormCheckbox,
-  },
-}
+
+Vue.use(FormCheckbox, {options});
```

Comment:
určite chceme globálne inštalovať? nie je to vhodné kvôli performance..
toto nie je žiaden plugin, určite by sme mali exportovať iba komponentu a nech si ju už developer includne tam kde treba, alebo keď je hlúpy tak si ju dá globálne.. nie ju registrovať napevno a ani nedáme možnosť si komponentu premenovať, ak už napr má komponentu s menom

Reply:
@padinko Globalne tam je prave kvoli validatorom a inym globalnym metodam. Suhlasim vsak ale s tym, ze by malo byt mozne vytiahnut komponent tam kde ho je treba. Na druhu stranu ten performance hit tam nebude kto vie aky...toto je malicka komponenta. Co sa zmeny nazvu tyce -> je na to option v plugin  optionoch. A tiez cez ten option mozes vlozit trans funkciu. Takze je to na zvazenie no.